src/cursor/cur_backup.c

```diff
@@ -299,12 +301,15 @@ __wt_curbackup_open(WT_SESSION_IMPL *session, const char *uri, WT_CURSOR *other,
         F_SET(cb, WT_CURBACKUP_QUERYID);
     }
 
+    /* Special backup cursor for export operation. */
+    export_backup = (strcmp(uri, "backup:export") == 0) ? true : false;
```

Comment:
It's safer to use `strncmp` with user input. Maybe it's also worth changing how we compare `backup:query_id` above.

Reply:
You should be using `WT_STRING_MATCH`.

Reply:
Right yeah, I was going off how we did the string comparison above but I'll change both to use `WT_STRING_MATCH`.